src/components/blocks/steps.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+  import type { StepsStoryblok } from '$types/bloks';
+  import Steps from '$components/pages/get-a-quote/steps.svelte';
+
+  export let block: StepsStoryblok;
+</script>
+
+<Steps page={block} variant="block" />
```

Comment:
`steps.svelte` should just be a unique file it's too confusing to nest components like this